Source/Plugins/CoherenceViewer/CumulativeTFR.cpp

```diff
@@ -75,7 +74,7 @@ void CumulativeTFR::addTrial(const double* fftIn, int chan)
 		// Multiple fft data by wavelet
 		for (int n = 0; n < nfft; n++)
 		{
-            freqData.set(n, freqData.getAsComplex(n) * waveletArray[freq][n]);
```

Comment:
This was a bug because each iteration of the outer loop modifies `freqData`, meaning that this call gets the wrong input after the first iteration.

The solution is to have a different buffer for the input to the IFFT than for the output of the FFT. To simplify, I decided to change the FFT and IFFT to be in-place - so we need just one `FFTWArray` for the forward transform and one for the inverse. Hope that makes sense.